addons/website/models/ir_ui_view.py

```diff
@@ -372,7 +372,6 @@ def _prepare_qcontext(self):
                 languages=request.env['res.lang'].get_available(),
                 translatable=translatable,
                 editable=editable,
-                menu_data=self.env['ir.ui.menu'].load_menus_root() if request.website.is_user() else None,
```

Comment:
should return {'chilfren':[]} to be retrocompatible

Reply:
done